contestantB/src/main/java/edu/asu/diging/contestantB/config/KafkaConsumerConfigs.java

```diff
@@ -0,0 +1,54 @@
+package edu.asu.diging.contestantB.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.common.serialization.IntegerDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.annotation.EnableKafka;
+import org.springframework.kafka.config.ConcurrentKafkaListenerContainerFactory;
+import org.springframework.kafka.core.ConsumerFactory;
+import org.springframework.kafka.core.DefaultKafkaConsumerFactory;
+
+@Configuration // this will tell Spring that this is a configuration class
+@EnableKafka  // this will tell the Spring-Kafka integration that you want to talk to Kafka
+
+public class KafkaConsumerConfigs {
+	public Map<String, Object> consumerConfigs() {
+	    Map<String, Object> props = new HashMap<>();
+	    // list of host:port pairs used for establishing the initial connections
+	    // to the Kakfa cluster
+	    props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG,
+	            "localhost:9092");
```

Comment:
this should be configurable. you don't want to change the class and recompile your project, just because kafka moved somewhere else ;)